decode_morse_message.rb

```diff
@@ -0,0 +1,39 @@
+def decode_char(char)
+  dictionary = { A: '.-', B: '-...', C: '-.-.', D: '-..',
+                 E: '.', F: '..-.', G: '--.', H: '....',
+                 I: '..', J: '.---', K: '-.-', L: '.-..',
+                 M: '--', N: '-.', O: '---', P: '.--.',
+                 Q: '--.-', R: '.-.', S: '...', T: '-',
+                 U: '..-', V: '...-', W: '.--', X: '-..--',
```

Comment:
- [ ] [OPTIONAL] the Morse character for the letter 'X' should be '-..-', because this wasn't noticed by the previous CR it isn't a requirement, however, I suggest you make this change as it is very small.